src/driver/cockroachdb/CockroachDriver.ts

```diff
@@ -618,7 +618,7 @@ export class CockroachDriver implements Driver {
                 || tableColumn.type !== this.normalizeType(columnMetadata)
                 || tableColumn.length !== columnMetadata.length
                 || tableColumn.precision !== columnMetadata.precision
-                || tableColumn.scale !== columnMetadata.scale
+                || (columnMetadata.scale && tableColumn.scale !== columnMetadata.scale)
```

Comment:
Do we actually want to do the first check?  What if there's a `scale` on the table column but not on the column metadata?  We don't want to remove it?

Or is this for an undefined value?  In that case, should this be `(typeof columnMetadata.scale === 'undefined' && tableColumn.scale !== columnMetadata.scale)`?  As `columnMetadata.scale = 0`  would short circuit it as written.

Reply:
I wrote it for undefined value. I missed it except mysql driver. Nice comment.